e2e/definitions/subject/pv.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: subject-pv
+  labels:
+    pv.kubernetes.io/type: target
+spec:
+  accessModes:
+  - ReadWriteMany
+  capacity:
+    storage: 1Gi
+  hostPath:
+    path: /var/local-path-provisioner/pvc-subject
```

Comment:
Where is the data being stored exactly? How do I inspect storage contents manually for debugging purposes or after e2e-testing?

Reply:
You can either `docker exec` into the kind controller container to check `/var/local-path-provisioner/pvc-subject` or you can `kubectl exec` into the `subject-container` pod and inspect the mount there.

Reply:
Any reasons against mounting a `testbin/data` folder directly? would be a better dev experience than exec' into containers